src/common/templates/includes/header-user.njk

```diff
@@ -18,33 +18,24 @@
       <a href="/" class="govuk-header__link govuk-header__link--service-name">{{__('title')}}</a>
 
       <button type="button" role="button" class="govuk-header__menu-button js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
+      {% macro nav_link(id, href, label) %}
+        <li id="nav-{{ id }}-li" class="govuk-header__navigation-item{{ ' govuk-header__navigation-item--active' if active == href}}">
+          <a id="nav-{{ id }}" class="govuk-header__link" href="{{ href }} ">{{ label }}</a>
+        </li>
+      {% endmacro %}
       <nav>
         <ul id="navigation" class="govuk-header__navigation" aria-label="Top Level Navigation">
-          <li class="govuk-header__navigation-item">
```

Comment:
The links are now created with the above macro to reduce the code. Macro also contains the logic for the active determination.